composer.json

```
@@ -26,8 +26,8 @@
     },
     "require": {
         "php": "^7.4 || ^8.0",
-        "contao/core-bundle": "^4.9",
-        "codefog/contao-haste": "^4.16"
+        "contao/core-bundle": "^4.9 || ^5.0",
```

```suggestion
        "contao/core-bundle": "^4.13 || ^5.0",
```

No point in allowing Contao 4.9 and requiring Haste 5 :)